controllers/publishController.js

```diff
@@ -7,23 +7,27 @@ module.exports = {
   publish (publishParams, fileName, fileType) {
     return new Promise((resolve, reject) => {
       let publishResults = {};
+      let file;
+      let claim;
       // 1. make sure the name is available
-      publishHelpers.checkNameAvailability(publishParams.name)
+      publishHelpers.checkClaimNameAvailability(publishParams.name)
```

Comment:
Does the daemon support two publishes to the same name if made from different channels?

Reply:
no, it will overwrite the first claim with the second, and just update the channel